src/statement/postgresql/update.js

```diff
@@ -11,12 +13,12 @@ class Update extends BaseUpdate {
    * @return Function            Returns `this`.
    */
   returning(fields) {
-    var fields = Array.isArray(fields) && arguments.length === 1 ? fields : Array.prototype.slice.call(arguments);
-    if (fields.length) {
-      this._parts.returning = this._parts.returning.concat(fields);
+    const arr = Array.isArray(fields) ? fields : [fields]
```

Comment:
I think this is effectively the same thing as calling slice on this. I don't see a need to do any king of cloning.
The docs state an Object or Array. So if it isn't an array, make an array of 1.

Reply:
:+1: